python/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -359,7 +359,7 @@ def master_prepare(self, app, appmasterconfig):
 
         if not isinstance(app.uploadedInput, DiracFile):
             generateDiracInput(app)
-            assert isinstance(app.uploadedInput, DiracFile), "Failed to upload needed file, aborting submit"
+            assert isinstance(app.uploadedInput, DiracFile), "Failed to upload needed file, aborting submit\nIf your Ganga installation is not at CERN your username may be trying to create a non-existent LFN. Try setting the 'DIRAC' configuration 'DiracLFNBase' to your grid user path.\n"
```

Comment:
This is suggesting using the DiracLFNBase configurable explicitly, rather than getting Ganga to piece it together from the username and VO. I thought it would be better to avoid possible conflicts elsewhere with the username config.